wrappers/swift/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.15)
+
+PROJECT(Swift_Wrapper)
+
+message("${CMAKE_CURRENT_SOURCE_DIR}")
+
+enable_language(Swift)
+
+# TODO: add logic to build dependencies
+
+include_directories(
+    ../../lib/include/public/
+    ../../lib/include/mat/
+)
+
+get_property(dirs DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY INCLUDE_DIRECTORIES)
+foreach(dir ${dirs})
+    message(STATUS "dir='${dir}'")
+endforeach()
+
+# Add source files to the target
+set(PLATFORM_FILES main.swift SDWEventProperties.swift)
+set(CMAKE_Swift_FLAGS "${CMAKE_Swift_FLAGS} -import-objc-header Swift_Wrapper-Bridging-Header.h")
+add_executable(swift_sample ${PLATFORM_FILES})
+target_link_libraries(swift_sample)
```

Comment:
This doesn't add any extra property, is it necessary?

Reply:
are you asking about the last line? it i used to create the final exe by linking all the libs.